_topic_maps/_topic_map_rosa.yml

```diff
@@ -563,6 +563,8 @@ Topics:
     File: rosa-nodes-about-autoscaling-nodes
   - Name: Configuring cluster memory to meet container memory and risk requirements
     File: nodes-cluster-resource-configure
+- Name: Working with nodes
```

Comment:
@eohartman Should this go in the cluster admin doc or in nodes? Something similar seems to have been commented out of nodes previously (see approx line 1260 of this file)

Reply:
I talked to Ben too, and he agreed it should be in the Nodes section.